developer/contributing/test-your-code.rst

```diff
@@ -0,0 +1,31 @@
+Test your Code
+=======================
+
+For running the tests, follow these steps:
+
+1. Install Composer dependences with ``composer install``.
+
+2. Run ``./bin/jackrabbit.sh``. This will install a local copy of Jackrabbit into the ``./bin`` directory and start the
+   server. If you already have a running server on your machine, you can skip this step. Also if you want the tests to run on
+   different port than the default `8080`, you need to install Jackrabbit on your own.
+
+3. Run the tests with ``./bin/runtests.sh``. This command has two possible options:
+
+   * ``-i``: Initialize the database and run all tests.
```

Comment:
`-i` just stands for initialization, you still have to either use the `-a` flag for running all tests or `-t` for specifying a single suite.